OWNERS

```diff
@@ -6,5 +6,4 @@ approvers:
   - saad-ali
   - xing-yang
   - wlan0
-  - brahmaroutu
-  - rrati
```

Comment:
In the future, please move approvers that are no longer active to "emeritus_approvers".